### DISCUSSION TITLE
cant copy paste url

### DISCUSSION THREAD

**Discussion by DigitalRama** [FreeTubeApp/FreeTube#5323] [General]
Hello there,

I just started using FreeTubes on Linux Nobara. I cannot seem to copy-paste URLs into the search bar while I wait for my Youtube data to export from google. Is this intended?

Many thanks

Rama

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Hi @DigitalRama it could be possible that the application will freezes momentarily when you import your data into it. This depends on how much it has to import.

**Comment by DigitalRama** [comment]
I'm not importing data yet. Sorry for the confusion. I am copying a URL from Firefox but it will not paste into FreeTube. it will paste into a text editor tho.

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Does this happen with one particular url or all urls?

**Reply by DigitalRama** [reply]
It worked one time then after that has not worked with any urls.

**Reply by efb4f5ff-1298-471a-8973-3d47447115dc** [reply]
Very werid im unable to reproduce this. What system did you install FreeTube on? Did you use an [Official download](https://github.com/FreeTubeApp/FreeTube/?tab=readme-ov-file#official-downloads) or an [Unofficial download](https://github.com/FreeTubeApp/FreeTube/?tab=readme-ov-file#unofficial-downloads)

**Reply by DigitalRama** [reply]
The system is Norbara Linux and it's Flapak Official Flathub - 0.21.0 Beta from the Software Center. I can try to install it from the GitHub if you would like.
